lib/container_server_linux.go

```diff
@@ -9,12 +9,16 @@ import (
 )
 
 func (c *ContainerServer) addSandboxPlatform(sb *sandbox.Sandbox) {
-	c.state.processLevels[selinux.NewContext(sb.ProcessLabel())["level"]]++
+	// NewContext() always returns nil, so we can safely ignore the error
+	ctx, _ := selinux.NewContext(sb.ProcessLabel())
+	c.state.processLevels[ctx["level"]]++
 }
 
 func (c *ContainerServer) removeSandboxPlatform(sb *sandbox.Sandbox) {
 	processLabel := sb.ProcessLabel()
-	level := selinux.NewContext(processLabel)["level"]
+	// NewContext() always returns nil, so we can safely ignore the error
+	ctx, _ := selinux.NewContext(processLabel)
```

Comment:
The same for here. Another option would be to revert the selinux-go related changes and put it in a separate PR like this: https://github.com/kubernetes-sigs/cri-o/pull/2127 WDYT? (If you disagree I will close my PR)